Графика для всех страниц

Kigasan opened the pull request:
---
:mortar_board: [Разметка всех страниц](https://up.htmlacademy.ru/htmlcss/38/user/2069137/tasks/5)

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/38/user/2069137/tasks/7)

Kigasan committed:
Добавил header для index

Kigasan committed:
Добавил hero и advantages

Добавил главную секцию и сразу за ней секцию преимуществ

Kigasan committed:
Добавил services и welcome

Kigasan committed:
Закончил разметку main

Изменил заголовок в welcome с b на h2, добавил секции для поиска отеля и форму для отправки e-mail

Kigasan committed:
Update index.html

Kigasan committed:
Добавил header и filter в catalog

Добавил фильтр на страницу каталога, ещё неизвестно что с формой, придется поменять, чтобы всё смотрелось как надо

Kigasan committed:
Добавил в catalog раздел о рассылке

Kigasan committed:
Добавил в каталог раздел сортировки

Kigasan committed:
Добавил в каталог карточки и pagination

Kigasan committed:
Внёс правки в index.html

Удалил один блок, скорректировал названия некоторых классов, убрал / из самозакрывающихся тегов

Kigasan committed:
Внёс правки в catalog.html

Изменил пути ссылок, раздел сортировки совместил с карточками в одну секцию, по мелочи поправил классы и убрал лишние заголовки

Kigasan committed:
Добавил href ссылкам

Добавил href ссылкам, ведущим на себя же, добавил контейнеры для обертки, поменял названия некоторых классов

Kigasan committed:
Добавил images

Добавил все изображения, подключил только контентные на обеих страницах

Kigasan committed:
for autocheck

изменения для автопроверки дз

Kigasan committed:
Исправил ошибки по стайлгайду с <img>

vlad-yermolayev commented on index.html:
```diff
@@ -5,8 +5,108 @@
+        <li class="socials-item"><a href="https://t.me/htmlacademy"><span class="visually-hidden">Telegram</span></a></li>
+        <li class="socials-item"><a href="https://www.youtube.com/user/htmlacademyru"><span class="visually-hidden">YouTube</span></a></li>
+      </ul>
+      <p class="footer-phone"><a href="tel:+78128121212">+7 (812) 812-12-12</a></p>
```
А зачем тебе эта лишняя обертка в виде `p`?

vlad-yermolayev commented on index.html:
```diff
@@ -5,8 +5,108 @@
+        <li class="socials-item"><a href="https://www.youtube.com/user/htmlacademyru"><span class="visually-hidden">YouTube</span></a></li>
+      </ul>
+      <p class="footer-phone"><a href="tel:+78128121212">+7 (812) 812-12-12</a></p>
+      <p class="footer-our-link"><a href="https://htmlacademy.ru/"><img src="images/htmlacademy-logo.svg" alt="логотип htmlacademy." width="114.9" height="32.9"></a></p>
```
А зачем тебе эта лишняя обертка в виде `p`?

vlad-yermolayev commented on index.html:
```diff
@@ -5,8 +5,108 @@
+            <a href="#"><span class="visually-hidden">Поиск</span></a>
+          </li>
+          <li class="nav-item">
+            <a href="#"><span class="visually-hidden">Избранные</span></a>
```
Добавь еще сюда пожалуйста `span` с числом добавленных в избранное отелей.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+      <li class="socials-item"><a href="https://www.youtube.com/user/htmlacademyru"><span class="visually-hidden">YouTube</span></a></li>
+    </ul>
+    <p class="footer-phone"><a href="tel:+78128121212">+7 (812) 812-12-12</a></p>
+    <p class="footer-our-link"><a href="https://htmlacademy.ru/"><img src="images/htmlacademy-logo.svg" alt="логотип htmlacademy." width="114.9" height="32.9"></a></p>
```
А зачем тебе эта лишняя обертка в виде `p`?

Kigasan replied:
Я думал, что стоит ссылки оборачивать в p. Действительно, наверное он там и не нужен, так что я уберу

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+      <li class="socials-item"><a href="https://t.me/htmlacademy"><span class="visually-hidden">Telegram</span></a></li>
+      <li class="socials-item"><a href="https://www.youtube.com/user/htmlacademyru"><span class="visually-hidden">YouTube</span></a></li>
+    </ul>
+    <p class="footer-phone"><a href="tel:+78128121212">+7 (812) 812-12-12</a></p>
```
А зачем тебе эта лишняя обертка в виде `p`?

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,212 @@
+          <a href="#" class="hotel-more">подробнее</a>
+          <button class="hotel-favorites-button" type="button">в избранное</button>
+          <p class="hotel-rating">
+            <img class="hotel-rating-image" src="images/stars-2.png" alt="2 звезды." width="140" height="37">
```
Это точно не контентное изображение. Лучше сделать через список, как я рассказывал на вебинарах.

Kigasan replied:
Обязательно посмотрю и исправлю!

Kigasan committed:
оптимизировал html

Изменил рейтинг с картинок на список, добавил span-ы в нужных местах, чтобы с сервера подбирать значения. Удалил лишние изображения звёздочек.
